diff --git a/ROMFS/px4fmu_common/init.d/airframes/CMakeLists.txt b/ROMFS/px4fmu_common/init.d/airframes/CMakeLists.txt
index 89498206c3..0ae1b1c3c4 100644
--- a/ROMFS/px4fmu_common/init.d/airframes/CMakeLists.txt
+++ b/ROMFS/px4fmu_common/init.d/airframes/CMakeLists.txt
@@ -93,6 +93,7 @@ if(CONFIG_MODULES_MC_RATE_CONTROL)
 
 		# [12000, 12999] Octo Cox
 		12001_octo_cox
+		22101_eolab_platypus
 
 		# [14000, 14999] MC with tilt
 		14001_generic_mc_with_tilt
@@ -156,4 +157,4 @@ if(CONFIG_MODULES_UUV_ATT_CONTROL)
 		60001_uuv_hippocampus
 		60002_uuv_bluerov2_heavy
 	)
-endif()
+endif()
\ No newline at end of file
